docker/ludwig-gpu/Dockerfile

```diff
@@ -35,7 +35,8 @@ RUN HOROVOD_GPU_OPERATIONS=NCCL \
     HOROVOD_WITHOUT_MPI=1 \
     HOROVOD_WITHOUT_TENSORFLOW=1 \
     HOROVOD_WITHOUT_MXNET=1 \
-    pip install --no-cache-dir '.[full]' && horovodrun --check-build
+    pip install --no-cache-dir '.[full]' --extra-index-url https://download.pytorch.org/whl/cu116 && \
```

Comment:
Is there an 11.3 wheel we can use? Does 11.6 wheel work with 11.3?

Reply:
I'm guessing the 11.6 wheel should be backwards compatible, but switching to the 11.3 wheel to be safe